docs/apis/cli.md

```diff
@@ -185,6 +189,18 @@ Action "list" lists running and scheduled programs.
      -s,--scheduled                Show only scheduled programs and their JobIDs
 
 
+Action "stop" stops a running program (streaming jobs only).
+
+  Syntax: stop [OPTIONS] <Job ID>
+  "stop" action options:
+     -m,--jobmanager <host:port>   Address of the JobManager (master) to which
+                                   to connect. Specify 'yarn-cluster' as the
+                                   JobManager to deploy a YARN cluster for the
+                                   job. Use this flag to connect to a different
```

Comment:
If you put "-m yarn-cluster", it will deploy a YARN cluster from the client. The "-m yarn cluster" option should only be available to the "run" command.
For stop, users should usually not specify the JobManager address, because
- in standalone mode, the JM address is in the conf
- in YARN Mode, the JM address is in a special `.yarn-properties` file.
Only if that is not working, users can use the "-m <jobmanager:port>".